packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -227,15 +227,12 @@ export class SummarizerNode implements IRootSummarizerNode {
 
             const props = {
                 summaryRefSeq,
-                pendingSize: this.pendingSummaries.size,
-                pendingHandle: this.pendingSummaries.size > 0 ? this.pendingSummaries.keys[0] : undefined,
-                pendingSeqNumber: this.pendingSummaries.size > 0 ?
-                    this.pendingSummaries.values[0].referenceSequenceNumber : undefined,
+                pendingSize: this.pendingSummaries.size ?? undefined,
             };
             this.defaultLogger.sendTelemetryEvent({
                 eventName: "PendingSummaryNotFound",
                 proposalHandle,
-                referenceSequenceNumber: this.referenceSequenceNumber,
+                referenceSequenceNumber: this.referenceSequenceNumber ?? undefined,
```

Comment:
why is this necessary? this.referenceSequenceNumber is always a number, no?  This would turn some number (like 0) to undefined?

Reply:
Indeed, it is not required - the error happened while trying to get the pending summary value.